Source/Scene/Model.js

```diff
@@ -1186,8 +1187,16 @@ define([
         var cacheKey = defaultValue(options.cacheKey, getAbsoluteUri(url));
 
         options = clone(options);
-        options.basePath = getBaseUri(url, true);
-        options.cacheKey = cacheKey;
+        if (defined(options.basePath)) {
```

Comment:
I don't think this would do what we want. We only want to append `basePath` to `cacheKey` if a cache key isn't specified in `options` by the user.